### DISCUSSION TITLE
shared objects in between workers in windows fastapi application with uvicron

### DISCUSSION THREAD

**Discussion by gudipudipradeep** [Kludex/uvicorn#1321] [Q&A] [answered]
shared objects in between workers in windows fastapi application with uvicron worker configuration.
@tomchristie  @graingert  Please suggest me how we can achieve this using uvicron fastapi application on windows platform.

**Comment by Kludex** [comment, marked as answer]
Not sure why, but you can check the python documentation about it: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Pipe
